truss/contexts/image_builder/serving_image_builder.py

```diff
@@ -91,7 +99,38 @@ def create_vllm_build_dir(config: TrussConfig, build_dir: Path):
     )
     nginx_template = read_template_from_fs(TEMPLATES_DIR, "vllm/proxy.conf.jinja")
 
-    dockerfile_content = dockerfile_template.render(hf_access_token=hf_access_token)
+    (build_dir / "cache_requirements.txt").write_text(spec.requirements_txt)
```

Comment:
Let's not do this. I think we should make a `cache_requirements.txt` in templates/ and just copy it directly. It should have the google client and huggingface hub

Reply:
add hf_transfer package to huggingface and set environment variable